packages/webchannel-wrapper/src/index.js

```diff
@@ -68,8 +68,6 @@ goog.net.XhrIo.prototype['getLastError'] =
 goog.net.XhrIo.prototype['getLastErrorCode'] =
   goog.net.XhrIo.prototype.getLastErrorCode;
 goog.net.XhrIo.prototype['getStatus'] = goog.net.XhrIo.prototype.getStatus;
-goog.net.XhrIo.prototype['getStatusText'] =
-  goog.net.XhrIo.prototype.getStatusText;
```

Comment:
This is not used by our code. Removing it is technically a breaking change though for consumers of this package. We might be ok though: https://github.com/search?l=JavaScript&q=%22%40firebase%2Fwebchannel-wrapper%22&type=Code